docs/organizations/metadata.mdx

```diff
@@ -5,47 +5,22 @@ description: Organization objects hold a set of metadata that can be used intern
 
 # Organization metadata
 
-Metadata allows for custom data to be saved on the `Organization` and `Organization Membership` objects. There are two types of metadata: **public** and **private**. Choosing which metadata field you should use for setting custom attributes on your organizations becomes a matter of access and visibility.
+Organization metadata allows you to store information about an organization that is not part of the standard fields. This can be useful for storing custom attributes that are specific to your application.
 
-| Metadata | Frontend API | Backend API |
-| --- | --- | --- |
-| private | - | Read and write |
-| public | Read | Read and write |
+There are two types of metadata: **public** and **private**.
 
-<Callout type="warning">
-    Metadata is limited to **8kb** maximum.
-</Callout>
+Both public and private metadata are set and can be accessed from the [Backend API](https://clerk.com/docs/reference/backend-api). Only public metadata can be accessed from the [Frontend API](https://clerk.com/docs/reference/frontend-api/).
 
-Both public and private metadata are set from the [Backend API](https://clerk.com/docs/reference/backend-api), but public metadata can be accessed from the [Frontend API](https://clerk.com/docs/reference/frontend-api/) and Backend API. These types of metadata can be used to access non-sensitive information about an organization, such as their profile picture or their name. They can be used to build UIs where an organization might not be signed in, but you still want to show some of their profile info.
+Both the `Organization` and `Organization Membership` objects have the metadata fields: `publicMetadata` and `privateMetadata`.
```

Comment:
```suggestion
Both the `Organization` and `Organization Membership` objects have the following metadata fields:
- `publicMetadata` –  Useful for setting metadata from your backend and have them displayed as read-only on the frontend.
- `privateMetadata` – Useful for setting custom attributes that contain sensitive information that should not be displayed on the frontend.
```

Combines this and the following lines for less repitition. Also, what do we mean by "displayed" here? Is `publicMetadata` accessible on the `Organization` object in the frontend, but `privateMetadata` is not? Or is it just expected that the developer won't display `privateMetadata` despite having access to it in the frontend?